packages/makeswift/src/init.ts

```diff
@@ -11,6 +11,7 @@ import { getProjectName } from './utils/get-name'
 import isNextApp from './utils/is-next-app'
 
 const MAKESWIFT_APP_ORIGIN = process.env.MAKESWIFT_APP_ORIGIN || 'https://app.makeswift.com'
+const MAKESWIFT_API_HOST = process.env.MAKESWIFT_API_HOST
```

Comment:
Can we change this variable to `MAKESWIFT_API_ORIGIN`. It used to be `MAKESWIFT_API_HOST` in the runtime but I realized that "host" was the wrong term and it should be "origin". I changed it in the runtime so we use `MAKESWIFT_API_ORIGIN` and `MAKESWIFT_APP_ORIGIN`

Reply:
Yeah of course. I was wondering why `MAKESWIFT_API_HOST` was having no effect

Reply:
Well, it will have no effect on the runtime right now, you'll need to pass options to the client as `new Makeswift(apiKey, { apiOrigin: process.env.MAKESWIFT_API_ORIGIN })`